Makefile

```diff
@@ -15,7 +15,7 @@ lint-diff:
 	git diff upstream/master --name-only -- "*.py" | xargs flake8
 
 black:
-	black . --exclude '(asv_bench/env|\.egg|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|_build|buck-out|build|dist|setup.py)'
+	black .
```

Comment:
Thoughts on removing this entirely?  The `make black` command is now longer than the actual `black .` command

Reply:
have *everything* available as a make command is quite powerful.